integration_tests/e2e/update-agree-plan.cy.ts

```diff
@@ -46,7 +46,7 @@ describe('Update Agree plan', () => {
     })
 
     it('Display update agree plan page correctly on load', () => {
-      cy.get('.plan-header+p').contains("Update Sam's agreement").click()
+      cy.get('#update-assessment-text').contains("Update Sam's agreement").click()
```

Comment:
does this work? I'd have thought it should be `'#update-assessment-text > a'` or something

Reply:
Yep this works 🙂

Reply:
but I mean the `click()` - there are two elements within the `#update-assessment-text`, the <a> and the inline text. Will `click()` always resolve correctly?